app/javascript/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Greeting from './Greeting'; 
+
+const App = () => { 
+  return (
+    <>
+    <Routes> 
+      <Route path="/" element={ <Greeting  />} />
+    </Routes>
+    </>
+  );
+}
+
+export default app;
```

Comment:
- [ ] When I Run the React app uses the URL to displays a random greeting I got the following error

- Error: Can’t resolve ‘react-router-dom’: This error is occurring because the `react-router-dom` module is not found in your project. You can install it by running the following command in your project’s root directory:

`npm install react-router-dom`

![image](https://github.com/DianaBeki/hello-rails-react/assets/93598189/92451ac2-e11f-4b55-a282-c2ecccac0977)